_vector-search/ai-search/hybrid-search/inner-hits.md

```diff
@@ -0,0 +1,927 @@
+                                      "description": "b, length normalization parameter",
+                                      "details": [
+                                        
+                                      ]
+                                    },
+                                    {
+                                      "value": 2.0,
+                                      "description": "dl, length of field",
+                                      "details": [
+                                        
+                                      ]
+                                    },
+                                    {
+                                      "value": 2.0,
+                                      "description": "avgdl, average length of field",
+                                      "details": [
+                                        
+                                      ]
+                                    }
+                                  ]
+                                }
+                              ]
+                            }
+                          ]
+                        }
+                      ]
+                    },
+                    {
+                      "value": 0.0,
+                      "description": "Not a match",
+                      "details": [
+                        
+                      ]
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        "inner_hits": {
+          "location": {
+            "hits": {
+              "total": {
+                "value": 0,
+                "relation": "eq"
+              },
+              "max_score": null,
+              "hits": [
+                
+              ]
+            }
+          },
+          "user": {
+            "hits": {
+              "total": {
+                "value": 2,
+                "relation": "eq"
+              },
+              "max_score": 0.31506687,
+              "hits": [
+                {
+                  "_index": "my-nlp-index",
+                  "_id": "2",
+                  "_nested": {
+                    "field": "user",
+                    "offset": 0
+                  },
+                  "_score": 0.31506687,
+                  "_source": {
+                    "name": "John Wick",
+                    "age": 46
+                  }
+                },
+                {
+                  "_index": "my-nlp-index",
+                  "_id": "2",
+                  "_nested": {
+                    "field": "user",
+                    "offset": 1
+                  },
+                  "_score": 0.31506687,
+                  "_source": {
+                    "name": "John Snow",
+                    "age": 40
+                  }
+                }
+              ]
+            }
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+## Sorting and Pagination with inner hits
```

Comment:
Do we also want to add one example for parent-join?

Reply:
Since as you mentioned earlier ingestion is not a focus here. So, the response format of nested query and parent-child will be same.